stores/sql/mysql/migrations/main/migration_00018_archived_contracts.sql

```diff
@@ -0,0 +1,139 @@
+DROP TABLE IF EXISTS contracts_temp;
+
+CREATE TABLE contracts_temp (
```

Comment:
Should be able to write a shorter migration for MySQL right? Without copying the data.